lib/sync_checker/formats/edition_base.rb

```diff
@@ -66,7 +66,8 @@ def checks_for_draft(locale)
               expected_details_hash(edition_expected_in_draft)
             end
           ),
-          Checks::TranslationsCheck.new(edition_expected_in_draft.available_locales)
+          Checks::TranslationsCheck.new(edition_expected_in_draft.available_locales),
+          Checks::HttpStatusCheck.new(edition_expected_in_draft)
```

Comment:
Yep. All of the other checks follow the convention of specifying what is expected. We seem to be unclear about which codes are expected, although we are clear on which codes should not be expected so maybe we could provide a range of disallowed response codes?

```rb
module SyncChecker
  module Checks
    HttpStatusCheck = Struct.new(:disallowed_response_codes) do
      def call(response)
        failures = []

        if (disallowed_response_codes).include?(response.response_code)
          failures << "http response code #{response.response_code} returned for #{response.request.base_url}"
        end

        failures
      end
    end
  end
end

HttpStatusCheck.new(400..599).call(response)
```